docs/source/model_design/programming_vivarium/running_simulations/index.rst

```diff
@@ -31,9 +31,207 @@ Running Simulations
+
+Match this information to the run specified in the vivarium research 
+documentation. 
+
+The scenarios File
+++++++++++++++++++
+
+The :code:`branches/scenarios.yaml` file contains information for when you 
+want to do multiple model runs. When we say multiple model runs - we mean 
+running on multiple draws/seeds/scenarios. Therefore we will use this on the 
+majority of our simulation runs. 
+
+The information in this file is limited. It contains the number of draws, seeds, 
+and scenarios to include in a simulation run. Check that this information 
+matches the run specified in the vivarium research documentation. 
+
+Running a Simulation on the Command Line
+----------------------------------------
+
+Once you've confirmed that your artifact is correct, and that your 
+:code:`model_spec.yaml` and :code:`branches/scenarios.yaml` files 
+are up to date, you're ready to run the model! 
+
+Creating an Environment
++++++++++++++++++++++++
+
+First, you will need to create an environment. This will be the same
+as creating usual environments, except when you install the project repo,
+you'll need to do a special editable install. The command looks like this :code:`pip install -e.[dev]`. Note that this is DIFFERENT than what is required for artifact building. 
+
+Additionally you'll need to install redis with :code:`conda install redis`.
+Some other packages can be helpful here as well like :code:`ipython` for
+debugging. 
+
+Generally, the project repo and redis are the only things you will need to install,
+but check with the engineers if there are updates to other vivarium
+packages you should be aware of - like :code:`vivarium_inputs`,
+:code:`vivarium_public_health`, or :code:`vivarium_gbd_access`.
+
+Simulate
+++++++++
+
+The :code:`simulate` run will run a single simulation based on the 
+:code:`model_spec.yaml` file. Since you're only running one simulation, 
+it won't look at the :code:`branches/scenarios.yaml` file at all. While 
+we mostly will need to use :code:`psimulate`, running a :code:`simulate` 
+run can be very helpful in debugging as it is much faster and will reveal 
+any bugs in the code. 
+
+Here is an example :code:`simulate` run and an explanation of the flags. 
+
+.. code-block:: bash 
+  :linenos:
+
+  $ simulate <PATH_TO_MODEL_SPEC> -vvv --pdb --proj_simscience_prod -o /mnt/team/simulation_science/pub/models/<PROJCET_NAME>/results/<MODEL_NUMBER>/ -i '<PATH_TO_ARTIFACT>' 
+
+Flags: 
+  - -vvv is for the verbosity, the vvv is standard on the team
+  - --pdb has you reach the python debugger if there are any errors
+  - --proj_simscience_prod has you run on the production project on the cluster, which has higher priority than other projects. You might need to be added to access this project! 
+  - -o is where to put the output results
+  - -i is the location of the artifact
+
+The simulate run should be relatively quick and won't take a tremendous amount 
+of data to run or store results. 
```

Comment:
```suggestion
of memory to run or store results.
```

Maybe? Not sure I totally understand this sentence.

Reply:
```suggestion
of space to run or store results. 
```
?